src/foundry/common/data/data.mjs/journalEntryPageData.d.ts

```diff
@@ -0,0 +1,290 @@
+import { ConfiguredFlags, FieldReturnType, PropertiesToSource } from "../../../../types/helperTypes";
```

Comment:
This file also will probably get refactored heavily but as the addition is necessary to get compilation to work at all I'll just get this file completely overridden. It's easier to override a set of changes like this than to get a clean diff in `journalEntryData`